src/entities/QuarterCategoryBudget/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { snakeCase } from 'lodash'
+
+import { TransparencyBudget } from '../../clients/DclData'
+import { NewGrantCategory } from '../Grant/types'
+import { QuarterBudgetAttributes } from '../QuarterBudget/types'
+
+export function toNewGrantCategory(category: string): NewGrantCategory {
```

Comment:
May I suggest this to avoid changing the code if a new category is added?

```typescript
function toNewGrantCategory(category: string) {
  const categories = Object.values(NewGrantCategory)
  const idx = categories.map(snakeCase).findIndex((value) => value === snakeCase(category))
  
  if(idx >= 0) {
    return categories[idx]
  }

  throw Error(`Attempted to parse an invalid NewGrantCategory ${category}`)
}
```